templates/taxi/manufacturer_list.html

```diff
@@ -2,6 +2,7 @@
 
 {% block content %}
     <h1>Manufacturer List
+      <a style="float: right" href="{%  url "taxi:manufacturer-create" %}" class="btn btn-secondary">ADD</a>
     </h1>
```

Comment:
Code Style Issue: According to the checklist, you need to use 2 whitespaces for indentation in your `.html` files. Here, it seems you are using 4 whitespaces for indentation.